front/app/routes.tsx

```diff
@@ -57,7 +58,16 @@ export type RouteType =
   | AdminRouteTypes
   | moduleRouteTypes
   | userShowPageRouteTypes
-  | citizenRouteTypes;
+  | settingsRegistrationRouteTypes
+  | citizenRouteTypes
+  | ExternalRouteTypes;
+
+type ExternalRouteTypes =
+  | `https${string}`
+  | `http${string}`
+  | `www${string}`
+  | `mailto${string}`
+  | `tel${string}`;
 
```

Comment:
These were the external routes I could think of - do you have any others? :thinking: